src/main/java/com/propertydekho/infoservice/InfoServiceApplication.java

```diff
@@ -1,9 +1,12 @@
 package com.propertydekho.infoservice;
 
 import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 
-@SpringBootApplication
+@SpringBootApplication(scanBasePackages = "com.propertydekho.infoservice")
+//@EnableAutoConfiguration(exclude = {DataSourceAutoConfiguration.class})
```

Comment:
Please remove the comment @Divsjains6021

Reply:
Resolved conflicts